iceoryx_posh/test/moduletests/test_popo_trigger.cpp

```diff
@@ -255,8 +325,10 @@ TEST_F(Trigger_test, ResetSetsTriggerIdToInvalid)
 
```

Comment:
Please check in ``InvalidateInvalidatesTrigger`` also the unique id, the trigger type ...